### DISCUSSION TITLE
Stuck in huggingface

### DISCUSSION THREAD

**Discussion by cakueayam** [lllyasviel/Fooocus#3351] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/e9257261-df7c-4001-bddc-a0330200e55f)

hi im stuck in this progress, what should i do?

**Comment by IPv6** [comment, marked as answer]
Try again later, happens sometimes. It either your internet connection problem or hugginface website slowdown

**Reply by cakueayam** [reply]
thank you very much!
